cli/internal/cmd/verify.go

```diff
@@ -191,16 +193,22 @@ func (c *verifyCmd) verify(cmd *cobra.Command, verifyClient verifyClient, config
 	switch c.flags.output {
 	case "json":
 		attDocOutput, err = formatJSON(cmd.Context(), rawAttestationDoc, attConfig, c.log)
+		if err != nil {
+			return fmt.Errorf("printing attestation document: %w", err)
+		}
+
 	case "raw":
 		attDocOutput = fmt.Sprintf("Attestation Document:\n%s\n", rawAttestationDoc)
+
 	case "":
 		attDocOutput, err = formatDefault(cmd.Context(), rawAttestationDoc, attConfig, c.log)
+		if err != nil {
+			return fmt.Errorf("printing attestation document: %w", err)
+		}
+
 	default:
 		return fmt.Errorf("invalid output value for formatter: %s", c.flags.output)
 	}
-	if err != nil {
-		return fmt.Errorf("printing attestation document: %w", err)
-	}
```

Comment:
This is functionally equivalent, right? Why did you decide to inline this?

Reply:
I think this way is cleaner, since not all cases set an error.
I would rather handle those errors in the case they originate.